app/bundles/ReportBundle/Model/ReportModel.php

```diff
@@ -239,15 +240,15 @@ public function buildAvailableReports($context)
                 $data[$context]['graphs'] = &$data['all']['graphs'][$context];
             } else {
                 //build them
-                $eventContext = ($context == 'all') ? '' : $context;
+                $eventContext = ('all' == $context) ? '' : $context;
```

Comment:
This seems to me that you have some custom CS Fixer or any other tool set?
Could you provide this to the community if so?

Reply:
I was using the mautic provided config file, contents pasted below. I noticed it tends to change things that mautic core has not been validated against yet. Is there a Mautic policy for php-cs-fixer, like when to or not to use it?

```
<?php

$finder = PhpCsFixer\Finder::create()
    ->in(__DIR__.'/app/bundles')
    ->in(__DIR__.'/app/config')
    ->in(__DIR__.'/app/middlewares')
    ->in(__DIR__.'/app/migrations')
    ->in(__DIR__.'/plugins');

return PhpCsFixer\Config::create()
    ->setRules([
        '@Symfony' => true,
        'binary_operator_spaces' => [
            'align_double_arrow' => true,
            'align_equals' => true
        ],
        'ordered_imports' => true,
        'array_syntax' => [
            'syntax' => 'short'
        ],
    ])
    ->setFinder($finder);
```

Reply:
This came from Symfony Coding standards, so Mautic should use it. However CS Fixer does not touch it for many devs. It is not checked on Travis too.